package.json

```diff
@@ -5,6 +5,8 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
+    "postinstall": "if test \"$NODE_ENV\" = \"production\" ; then npm run build ; fi",
+    "start": "node server.js",
```

Comment:
These commands should allow for continuous deployment to heroku

Reply:
Wow that first one is a beast